fire/SFMainMod.F90

```diff
@@ -306,6 +306,10 @@ subroutine  charecteristics_of_fuel ( currentSite )
           currentPatch%litter_moisture(tr_sf)       = fuel_moisture(tr_sf)/MEF(tr_sf)
           currentPatch%litter_moisture(dl_sf)       = fuel_moisture(dl_sf)/MEF(dl_sf)
           currentPatch%litter_moisture(lg_sf)       = fuel_moisture(lg_sf)/MEF(lg_sf)
+
+          ! remove trunks from patch%sum_fuel because they should not be included in fire equations
+          ! NOTE: ACF will update this soon to be more clean/bug-proof
+          currentPatch%sum_fuel = currentPatch%sum_fuel - litt_c%ag_cwd(4)
```

Comment:
Would be safer to use tr_sf rather than 4, no?

Reply:
well....`tr_sf` is the trunks index in the SPITFIRE fuel array, not the litter array... though they are (implicitly) set up so that they are supposed to be the same order so I guess yes? I was going to fix this in the refactor